### ISSUE TITLE
Alternative Searchkick repo

### ISSUE THREAD

**Issue by nmella** [marketshop-io/spree_searchkick#1]
Hi @rahul-bluebash 

Shouldn't we have forked this repo instead ?

https://github.com/spree-contrib/spree_searchkick

It seems more updated...

**Comment by rahul-bluebash** [comment]
@nmella Yes its is more updated and compatible too with spree latest version. Earlier I was thinking to use this repo, while I was upgrading spree and rails.
The reason we chose old repo was because - 
- That repo was already used in our current project with customisation, I think it was kept it our project directory when it project started by Vishal.
- There was lot of customisation made as per the old extension, switching to this new one, might require additional work. 

Ideally, we should use this repo and migrate all customisation to this gem. So you can fork this repo too.

**Comment by nmella** [comment]
OK, thanks. Yes, I prefer to use this repo too. I will fork it. 

I think we should focus on the masterbase project as a new product for **new projects mainly**, without thinking on our current projects (mercadotech, mercadodeportivo). I think we might never switch our current projects to use the masterbase product, as this might be too complex, so we can keep them as it is. 

Thanks. 

PS: When trying to fork [the new](https://github.com/spree-contrib/spree_searchkick) repo, it says we already have the other "spree_searchkick" forked so I can't fork it. I need to delete this repo in order to fork the new one. So, should I delete the old repo and for the new one ?

**Comment by rahul-bluebash** [comment]
@nmella Yes, please delete this, and add new one.
